official/nlp/keras_nlp/layers/transformer_encoder_block.py

```diff
@@ -181,7 +177,8 @@ def build(self, input_shape):
         name="output",
         kernel_initializer=self._kernel_initializer,
         **common_kwargs)
-    self._output_dropout = tf.keras.layers.Dropout(rate=self._output_dropout)
+    self._output_dropout_layer = tf.keras.layers.Dropout(
+      rate=self._output_dropout)
```

Comment:
Please add two spaces before "rate=self._output_dropout)", just to keep the coding style consistent.

Reply:
Same as before.